modules/sr/robot/motor.py

```diff
@@ -36,9 +40,14 @@ def translate(sr_speed_val: int, sr_motor: Union[Gripper, Wheel, LinearMotor]) -
 class Motor:
     """Represents a motor board."""
 
-    def __init__(self, m0: Union[Wheel, LinearMotor], m1: Union[Wheel, Gripper]) -> None:
+    def __init__(
+        self,
+        m0: Union[Wheel, LinearMotor, Gripper],
+        m1: Union[Wheel, LinearMotor, Gripper, None],
```

Comment:
Is there a reason the `m0` is not optional?

Reply:
My thinking was that there wasn't a use for a motor board with no motors attached so you would always be using the first motor.

Reply:
Would we ever want to create a motorboard with `m1` but not `m0`? Thinking about the physical kit its possible for this to happen with teams just plugging one motor into `m1`. I think it'd be nice from a future proofing point of view if we let either of them be None. But saying that as we're designing the robots in the simulator I don't think we'd ever want to skip m0. I'll leave up to you :)

Reply:
You make a good point, as there doesn't seem to be any downside I've made `m0` optional in 06a5e045c8784f0361226bc6625410087eaf7ad0